server/routes/createGoal/createGoalController.test.ts

```diff
@@ -57,6 +58,7 @@ describe('createGoalController', () => {
         'targetDate-year': '2024',
         action: 'submit-form',
       }
+      req.session.addStepForms = []
 
       mockedValidateAddStepForm.mockReturnValue([])
 
```

Comment:
(The stupid diff wont let me add a comment where I want to!), but is it worth adding an assertion in the `// Then` block about the `addStepForms` array in the session?
Presumably we expect the `addStepForms` array to have had a new step added to it?
Something like:

```
// Then
...
...
expect(req.session.addStepForms).toEqual(....)
```